js/libs/keycloak-masthead/src/KeycloakContext.tsx

```diff
@@ -0,0 +1,33 @@
+import Keycloak, { KeycloakTokenParsed } from "keycloak-js";
+import { createContext, PropsWithChildren, useContext, useState } from "react";
+
+type KeycloakProps = {
+  keycloak?: Keycloak;
```

Comment:
Under which circumstances would the Keycloak instance not be defined? Perhaps this can be made required?
```suggestion
  keycloak: Keycloak;
```